lhotse/features/base.py

```diff
@@ -446,6 +446,7 @@ def load(
         self,
         start: Optional[Seconds] = None,
         duration: Optional[Seconds] = None,
+        channel_id: Union[int, List[int]] = 0,
```

Comment:
This should address https://github.com/lhotse-speech/lhotse/issues/908, but it is currently unused in the function.